R/xml_text.R

```diff
@@ -109,7 +118,7 @@ xml_double.xml_node <- function(x) {
 
 #' @export
 xml_double.xml_nodeset <- function(x) {
-  vapply(x, xml_double, numeric(1))
+  as.numeric(xml_text(x))
```

Comment:
Maybe this could just become the implementation for `xml_double()`? (And similarly for `xml_integer()`)